Zadatak2.java

```diff
@@ -0,0 +1,40 @@
+package homework1;
+
+import java.util.Scanner;
+
+public class Zadatak2 {
+
+	public static void main(String[] args) {
+		int a;
+		int b;
+		String operacija = "";
+
+		Scanner ulaz = new Scanner(System.in);
+		System.out.println("Unesite dva cela broja.");
+
+		a = ulaz.nextInt();
+		b = ulaz.nextInt();
+
+		System.out.println("Unesite operaciju (�s� � sabiranje, �o� � oduzimanje, �m� � mno�enje, �d� � deljenje, �e� - izlaz).");
+
+		operacija = ulaz.next();
+
+		double rezultat = 0;
+
+		switch (operacija) {
+		case "s":
+			rezultat = a + b;
+			break;
+		case "o":
+			rezultat = a - b;
+			break;
+		case "m":
+			rezultat = a * b;
+			break;
+		case "d":
+			rezultat = a / b;
+			break;
+		}

```

Comment:
u sustini korektno, jedino mi je zasmetalo ovde sto ne postoji default grana, i sto ti se ispise na ekranu "Rezultat je: 0.0" kad uneses nepostojecu operaciju.
ja sam to resio ovako:
switch(operacija) {
	case 's': System.out.println("Rezultat je: " + (broj1 + broj2));
		break;
	case 'o': System.out.println("Rezultat je: " + (broj1 - broj2));
		break;
	case 'm': System.out.println("Rezultat je: " + (broj1 * broj2));
		break;
	case 'd': System.out.println("Rezultat je: " + (broj1 / broj2));
		break;
	default: System.out.println("Nepostojeca operacija!");

Reply:
Aha, da, tek sto sam pocela da ucim Javu, a vec sam zaboravila da razmisljam kao tester. 😄 
Hvala, biti ce poboljsano.